frontend/src/types/store.ts

```diff
@@ -196,23 +198,31 @@ export interface SqlEditorState {
   shouldSetContent: boolean;
   queryHistoryList: QueryHistory[];
   isFetchingQueryHistory: boolean;
-  savedQueryList: SavedQuery[];
-  isFetchingSavedQueries: boolean;
   isExecuting: boolean;
   isShowExecutingHint: boolean;
 }
 export type SqlEditorGetters = typeof sqlEditorStore.getters;
 export type SqlEditorActions = typeof sqlEditorStore.actions;
 export type SqlEditorMutations = typeof sqlEditorStore.mutations;
 
-export interface EditorSelectorState {
-  queryTabList: TabInfo[];
+export interface TabState {
+  tabList: TabInfo[];
   activeTabId: string;
 }
-export type EditorSelectorGetters = typeof editorSelectorStore.getters;
-export type EditorSelectorActions = typeof editorSelectorStore.actions;
-export type EditorSelectorMutations = typeof editorSelectorStore.mutations;
+export type TabGetters = typeof tabStore.getters;
+export type TabActions = typeof tabStore.actions;
+export type TabMutations = typeof tabStore.mutations;
 
 export interface DeploymentState {
   deploymentConfigByProjectId: Map<ProjectId, DeploymentConfig>;
 }
+
+export interface SheetState {
+  sheetList: Sheet[];
+  sheetById: Map<SheetId, Sheet>;
+  sharedSheet?: Sheet | null;
```

Comment:
Actually, I am not sure why we need to keep a sharedSheet state, please add comment